examples/facets_visualize_statistics/run.py

```diff
@@ -119,7 +122,7 @@ def visualize_statistics():
     # Run the pipeline
     p = boston_housing_pipeline(
         importer=importer(),
-        trainer=trainer(),
+        trainer=trainer().with_return_materializers(KerasMaterializer),
```

Comment:
Do we need this here?

Reply:
sorry i will remove it, we don't need it actually